Dalamud/Interface/Internal/Windows/Data/Widgets/IconBrowserWidget.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using System.Collections.Generic;
-using System.Drawing;
-using System.Linq;
 using System.Numerics;
+using System.Threading.Tasks;
```

Comment:
I think someone else was intending to make a full-featured icon browser but was waiting for this PR, so I think it's best to leave it internal for now so that that other PR won't have any compatibility constraints.

Reply:
This one was intended to be a more "raw" icon browser that shows all the icons without any filtering or categorization. QoL's icon browser suffered from over filtering when new icons were added to the game. The browser that someone was wanting to make categorizes them manually.